ansible/roles/os_config/tasks/network_optimization.yml

```diff
@@ -0,0 +1,15 @@
+- name: update sysctl net.ipv4.neigh.default.gc_thresh1
```

Comment:
i`m not sure this needs to be done .. are there any downsides to increasing this ?

Reply:
@LazyDBA247-Anyvision.

Reply:
@sashas-anyvision @OriBenHur-Any 
we not sure we saw this bug again, or if that was the main cause, but that the setting i changed on a not working system, and 10 seconds after v2+insights got to healthy/working.
but i am sure the change won't break anything, so "better safe than sorry?"